.ci/ppc64le/lib_install_qemu_ppc64le.sh

```diff
@@ -42,5 +42,8 @@ build_and_install_qemu() {
 
         sudo ln -sf $(command -v ${BUILT_QEMU}) "/usr/bin/qemu-system-${QEMU_ARCH}"
 
+        # Check if virtiofsd is properly installed
+        ls -l /usr/libexec/qemu/virtiofsd || return 1
```

Comment:
thank you @Amulyam24 , could you manage this check for example using an if statement:
```
if [ ! -f /dir/foo.txt ]; then 
    echo "INFO: an error message"
    return 1 
fi
```

Reply:
@dborquez, thanks for the suggestion. Addressed it, PTAL!